tests/bearlib/abstractions/LintTest.py

```diff
@@ -126,6 +128,12 @@ def test_check_prerequisites(self):
         Lint.prerequisite_command = ["command_which_doesnt_exist"]
         self.assertEqual(Lint.check_prerequisites(), Lint.prerequisite_fail_msg)
 
+        Lint.prerequisite_command = "command_which_isnt_a_list"
+        Lint.prerequisite_fail_msg = "'{}' should be a list.".format(
```

Comment:
I do not understand why you're setting this prerequisite fail msg here. Cant you directly check it
`self.assertEqual(Lint.check_prerequisites(), "'{}' should be a list.".format(Lint.prerequisite_command))` ?